tests/providers/aws/lib/mutelist/aws_mutelist_test.py

```diff
@@ -1844,3 +1844,37 @@ def test_is_muted_in_resource_starting_by_star(self):
         allowlist_resources = ["*.es"]
 
         assert AWSMutelist.is_item_matched(allowlist_resources, "google.es")
+
+    def test_mute_finding(self):
+        # Mutelist
+        mutelist_content = {
+            "Accounts": {
+                AWS_ACCOUNT_NUMBER: {
+                    "Checks": {
+                        "check_test": {
+                            "Regions": [AWS_REGION_US_EAST_1, AWS_REGION_EU_WEST_1],
+                            "Resources": ["prowler", "^test", "prowler-pro"],
+                        }
+                    }
+                }
+            }
+        }
+        mutelist = AWSMutelist(mutelist_content=mutelist_content)
+
+        # Finding
+        finding_1 = MagicMock
```

Comment:
Why are you not using a `Finding`? We have a cool generator for this.

Reply:
Done